pyrobosim/examples/demo.py

```diff
@@ -27,10 +27,14 @@ def create_world(multirobot=False):
     """Create a test world"""
     world = World()
 
-    # Set the location and object metadata
     world.set_metadata(
-        locations=os.path.join(data_folder, "example_location_data.yaml"),
-        objects=os.path.join(data_folder, "example_object_data.yaml"),
+        locations=os.path.join(data_folder, "example_location_data_furniture.yaml"),
+        objects=os.path.join(data_folder, "example_object_data_food.yaml"),
+    )
+
+    world.add_metadata(
+        locations=os.path.join(data_folder, "example_location_data_accessories.yaml"),
+        objects=os.path.join(data_folder, "example_object_data_drink.yaml"),
     )
```

Comment:
While `set_metadata()` following by arbitrarily many `add_metadata()` works fine, I think for consistency let's just use two `add_metadata()` calls.

Reply:
Actually, this makes me think whether we should support passing lists in `locations` and `objects` args so the calls aren't so redundant.

I'm specifically thinking of the case where you may e.g. have 1 location metadata file and like 10 object metadata files.

Reply:
Oh also -- make sure all these extra metadata are also included in the ROS version of the example:

https://github.com/sea-bass/pyrobosim/blob/92bbe059d6c437c536e8e322b7a4fe3f3f433c5c/pyrobosim_ros/examples/demo.py#L27-L31